src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
You don't need to reset default paddings here, also * selector is only suitable for resetting box-sizing, any other properties should be put in explicit tags selector, it your case it's only the body